lib/js/components/Banner/Banner.vue

```diff
@@ -197,6 +221,39 @@
 		padding: $space-xxxxs 0;
 	}
 
+	&__rightWrapperHorizontal {
```

Comment:
don't we need to vertically align items in this wrapper. When I type some text, it's positioned higher than button

Reply:
the same for vertical

Reply:
I changed the vertical/horizontal display, no duplicate code any more [`89e1e65` (#193)](https://github.com/bethinkpl/design-system/pull/193/commits/89e1e657b51a65c9a777bab702e67133da5ecaae)